cobalt/BUILD.gn

```diff
@@ -43,6 +43,7 @@ if (!is_android) {
     defines = []
 
     deps = [
+      ":common",
```

Comment:
im assuming this was added as gn check failed for this target ?

Reply:
Yeah